recipes/pip/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version="8.1.2" %}
+
+package:
+  name: pip
+  version: {{ version }}
+
+source:
+  fn: pip-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pip/pip-{{ version }}.tar.gz
+  md5: 87083c0b9867963b29f7aba3613e8f4a
+
+build:
+  number: 0
+  entry_points:
+    - pip = pip:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
+    - wheel
+
+test:
+  commands:
+    - pip -h
+    - pip list
+  imports:
+    - pip
+
+about:
+  home: https://pip.pypa.io/en/stable/
+  license: MIT
+  license_file: LICENSE.txt
+  summary: PyPA recommended tool for installing Python packages
+  doc_url: https://pip.pypa.io/en/stable/
+  dev_url: https://github.com/pypa/pip
+
+extra:
+  recipe-maintainers:
+    - msarahan
```

Comment:
Maybe we should add ourselves here too, @jjhelmus? I'm guessing the usual gang is interested @pelson, @ocefpaf, yourself, and me. Though I'm ok with doing this in the feedstock too. Either way.